core/src/test/java/com/linecorp/armeria/server/TestConverters.java

```diff
@@ -38,7 +38,7 @@ public HttpResponse convertResponse(ServiceRequestContext ctx,
                                             @Nullable Object result,
                                             HttpHeaders trailers) throws Exception {
             if (result instanceof Integer) {
-                return httpResponse(HttpData.ofUtf8(String.format("Integer: %d", result)));
+                return httpResponse(HttpData.ofUtf8(String.format("Integer: %d", (Integer) result)));
```

Comment:
Isn't this cast redundant?

Reply:
Yeah, it's redundant but it seems that the Errorprone couldn't recognize and indicates this is an error. 😱

Reply:
It thinks it's a String. 😱

Reply:
In Java 14 maybe it doesn't complain? :) But yeah seems like a false positive but harmless workaround